.github/workflows/release-java.yml

```diff
@@ -0,0 +1,34 @@
+name: Release maven package
+
+on:
+  push:
+    branches: [release/*]
+
+jobs:
+  pre-release-check:
+    uses: cucumber/.github/.github/workflows/prerelease-checks.yml@main
+
+  test-java:
+    uses: ./.github/workflows/test-java.yml
+
+  publish-mvn:
+    name: Publish Maven Package
+    needs: [pre-release-check, test-java]
+    runs-on: ubuntu-latest
+    environment: Release
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v2
+        with:
+          distribution: "temurin"
```

Comment:
should be openjdk

Reply:
Not supported by: https://github.com/actions/setup-java

Consider switching everything to azul.

Reply:
The release action will also need a `toolchains.xml`.

Reply:
Or maybe use 
```
      - uses:
          AdoptOpenJDK/install-jdk@v1
```

Reply:
Resolved by not depending on a vendor in https://github.com/cucumber/cucumber-jvm/pull/2595